jkube-kit/generator/webapp/src/main/java/org/eclipse/jkube/generator/webapp/WebAppGenerator.java

```diff
@@ -150,9 +156,24 @@ private AppServerHandler createCustomAppServerHandler(String from) {
     Map<String, String> defaultEnv = new HashMap<>();
     defaultEnv.put("DEPLOY_DIR", getDeploymentDir(handler));
     defaultEnv.putAll(handler.getEnv());
+
+    defaultEnv.putAll(extractEnvVariables(getConfig(Config.ENV)));
+
     return defaultEnv;
   }
 
+  public static Map<String, String> extractEnvVariables(String envConfigValue) {
+    if (StringUtils.isBlank(envConfigValue)) {
+      return Collections.emptyMap();
+    }
+
+    return Pattern.compile("\\s+").splitAsStream(envConfigValue) // "envName1=value1 envName2=value2"
+        .map(envNameValue -> envNameValue.split("=")) //
```

Comment:
nit, are these comments necessary?

Reply:
That's my way to tell the formatter to "format and go to line there !"
But I can remove these `//` if it's a problem.